src/SUMMARY.md

```diff
@@ -77,6 +77,7 @@ Robust Toolbox
     - [Entities](en/robust-toolbox/toolshed/commands/entity-control.md)
     - [General](en/robust-toolbox/toolshed/commands/general.md)
     - [Miscellaneous](en/robust-toolbox/toolshed/commands/misc.md)
+  - [Tollshead Examples](en/robust-toolbox/toolshed/toolshead-examples.md)
```

Comment:
"Toolshed" is the preferred spelling

Reply:
Check the filename, too

Reply:
Oh god, I see it now. I hate my dyslexia.